website/source/docs/guides/acl-replication.md

```diff
@@ -0,0 +1,244 @@
+---
+name: 'ACL Replication for Multiple Datacenters '
+content_length: 15
+id: acl-replication
+layout: content_layout
+products_used:
+  - Consul
+description: You can configure tokens, policies, and roles to work across multiple datacenters.
+---
+
+You can configure tokens, policies, and roles to work across multiple
+datacenters. This provides central configuration of security policies in the
+primary datacenter and failover for the secondary datacenters. If the primary
+datacenter goes offline, the secondary datacenters will still have the security
+policies in place. 
+
+Note, each datacenter should have ACLs enabled and you need to complete the
+bootstrapping process outlined in the [Securing Consul with ACLs
+guide](/consul/security-networking/production-acls) on the primary datacenter.
+This guide includes the additional ACL replication configuration for the Consul
+agents. 
+
+In this guide, you will setup ACL replication. this is a multi-step process
+that includes:
+
+Setting the `primary_datacenter` parameter on all Consul agents in the primary
+datacenter.  Creating the replication token.  Configuring the
+`primary_datacenter` parameter on all Consul agents in the secondary
+datacenter.  Enabling token replication on the servers in the secondary
+datacenter.  Applying the replication token to all the servers in the secondary
+datacenter. 
+
+This guide is intended to be completed during the initial ACL bootstrapping
+process. 
+
+-> After ACLs are enabled you must have a privileged token to complete any
+operation on either datacenter. The privileged token can be the initial
+`bootstrap` token. 
+
+## Configure the Primary Datacenter
+
+~> Note, if your primary datacenter uses the default `datacenter` name of
+`dc1`, you must set the `datacenter` parameter on the secondary datacenter.
+Otherwise, both datacenters will be named `dc1` and there will be conflicts.
```

Comment:
I think we should make wan-joining the datacenters together a pre-guide to this guide. That's the primary place the `datacenter` name is used, and you'll be issuing wan joining commands (or using the equivalent in consul configs). It might be weird to inject a bunch of content about network topology/connectivity setup into the guide about configuring ACLs.

Reply:
This does seem like a good idea. This guide should probably assume you know how to join datacenters and know how to enable ACLs in a single DC. This guide then is how to combine both concepts.

Reply:
Agreed, it is out of place in this guide. I will remove the parameter and update guide.